src/applications/personalization/dashboard-2/tests/unit/healthcare.unit.spec.jsx

```diff
@@ -5,18 +5,48 @@ import { expect } from 'chai';
 import { HealthCare } from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
 
 describe('HealthCare', () => {
-  describe('Manage all your prescriptions', () => {
-    const defaultProps = {
-      appointments: [],
-      authenticatedWithSSOe: true,
-      canAccessRx: false,
-      fetchConfirmedFutureAppointments: () => {},
-    };
+  const defaultProps = {
+    appointments: [],
+    authenticatedWithSSOe: true,
+    canAccessRx: false,
+    canAccessMessaging: true,
+    fetchFolder: () => {},
+    fetchConfirmedFutureAppointments: () => {},
+  };
 
-    it('when enrolled in VA health care, should not render when they have never had any prescriptions managed by the VA', () => {
-      const wrapper = mount(<HealthCare {...defaultProps} />);
-      expect(wrapper.find('Prescriptions').exists()).to.be.false;
-      wrapper.unmount();
+  describe('Prescriptions', () => {
+    describe('when enrolled in VA Health Care', () => {
+      it('should not render when they have never had any prescriptions managed by the VA', () => {
+        const wrapper = mount(<HealthCare {...defaultProps} />);
+        expect(wrapper.find('Prescriptions').exists()).to.be.false;
```

Comment:
This is an example of why I'm not a fan of Enzyme because it's testing an implementation detail.

Let's say in the future the Prescriptions component is not directly rendered by the HealthCare component. Maybe the structure of the JSX changes in such a way that: HealthCare renders NewWrapperComponent which renders Prescriptions. This test will now fail.

I've run into breaking tests like this twice in the past month or so.

The RTL approach would be to render the HealthCare component and make sure that content _rendered by_ Prescriptions does not exist in the DOM.